Neos.Media/Classes/ViewHelpers/FileTypeIconViewHelper.php

```diff
@@ -75,7 +75,9 @@ public function initializeArguments()
      */
     public function render(AssetInterface $file = null, AssetInterface $asset = null, string $filename = null, $width = null, $height = null)
     {
-        $asset = $file;
+        if (!is_null($file)) {
```

Comment:
Mhm with that, if you give both $asset and $file, the deprecated file would take precedence.

Nitpick using `$file !== null` is more common in the Neos code.

I would recomend:

```
if($asset === null) {
  $asset = $file;
}
```

Reply:
Thank you
 - shall I adapt the code to your solution **HERE** ... 
(another commit, no ?)
**OR** create a new PullRequest ? (squash)